README.md

```diff
@@ -289,21 +289,21 @@ The Exploits API provides access to several exploit/ vulnerability data sources.
 Search across a variety of data sources for exploits and use facets to get summary information.
 
 ```ruby
-client.search("python")             # Search for Snek vulns.
-client.search(post: 22)             # Port number for the affected service if the exploit is remote.
-client.search(type: "shellcode")    # A category of exploit to search for.
-client.search(osvdb: "100007")      # Open Source Vulnerability Database ID for the exploit.
+client.exploits_api.search("python")             # Search for Snyk vulns.
```

Comment:
This is actually a reference to the [snek](https://knowyourmeme.com/memes/snek) meme. 😅